src/components/screens/monitor/blocks/blocks.js

```diff
@@ -53,7 +53,7 @@ const Blocks = ({
           <BlockFilterDropdown filters={filters} applyFilters={applyFilters} />
         </BoxHeader>
         <LoadLatestButton
-          event="update.block"
+          entity="block"
```

Comment:
Can we make this "block" string a constant and import it? That would be more scalable and not prone to crash.

Reply:
I do agree we should prevent mistakes there, but I think propTypes is a better choice than constants.